ppr-ui/src/components/mhrRegistration/YourHome/OtherInformation.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div>
+    <v-card flat class="white pb-6 pt-6 pr-10 pl-8 rounded">
```

Comment:
`pb` and `pt` can be shortened to `py` if the px count is the same.
`py` and `px` is padding top/bottom, left/right respectively.

Reply:
Shortened.